index.html

```diff
@@ -0,0 +1,193 @@
+        <div class="container">
+          <a class="navbar-brand" href="#">
+                <img src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" width="30%" height="60%" alt="" id="header-img">
+              </a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+          <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="#features">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#works">How it works</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#pricing">Pricing</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <div class="row" style="padding-top: 80px;">
+      <div class="col-12">
+        <h4 style="text-align: center;">
+          <b>Handcrafted, home-made masterpieces</b>
+        </h4>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-4"></div>
+      <div class="col-4">
+        <form id="form" method="POST" action="https://www.freecodecamp.com/email-submit">
+          <div class="form-group">
+            <input type="email" class="form-control" name="email" id="email" aria-describedby="emailHelp" placeholder="Enter email" required>
+          </div> <p></p><p></p>
+          <div class="form-group">
+            <div style="text-align: center;">
+              <input type="submit" id="submit" class="btn btn-warning text-center"  value="GET STARTED"/>
+            </div>
+          </div>
+        </form>
+      </div>
+      <div class="col-4"></div>
+    </div> 
+    <div class="row" style="padding-top: 100px;">
+      <div class="row">
+          <div class="col-4" style="padding-right: 90px;">
+            <i class="fas fa-fire" style="font-size:xxx-large; color: orange; float: right;"></i>
+
+          </div>
+          <div class="col-7">
+              <div>
+                <h4><b> Premium Materials</b></h4>
+                Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
+              </div>
+          </div>
+      </div> 
+      <a name="features">.</a>
+      <div class="row" style="padding-top: 70px;" id="features">
+        <div class="col-4" style="padding-right: 80px;">
+          <i class="fas fa-truck"  style="font-size:xxx-large; color: orange; float: right;"></i>
+        </div>
+        <div class="col-7">
+            <div>
+              <h4><b> 
+                Fast Shipping</b></h4>
```

Comment:
Nice ideas on positioning, but please can you wrap this section I highlighted is wrapped with two div tags and a flexbox to sort out the positioning?. You can also go ahead and use CSS to sort out the font-weight you are trying to achieve with `h4` and `b`. 😇

This 👆 is better HTML semantics and use of tags

Reply:
Nice comment, well explained and human-friendly due to the use of emojis